### PR TITLE
feat(migrate): display datasource info in CLI

### PR DESCRIPTION
**Pull request by Jolg42**
Closes https://github.com/prisma/migrations-team/issues/125

### CODE REVIEW

File: src/packages/migrate/src/__tests__/MigrateDev.test.ts [hunk 1]
```diff
@@ -573,14 +594,15 @@ describe('sqlite', () => {
 
     await expect(result).rejects.toMatchInlineSnapshot(`
 
-                                                                                                                                                                                                                                                                                    ⚠️ We found changes that cannot be executed:
+                                                                                                                                                                                                                                                                                                ⚠️ We found changes that cannot be executed:
```
Comment (williamluke4) [inline src/packages/migrate/src/__tests__/MigrateDev.test.ts:597]: Is there meant to be this much whitespace?

File: src/packages/migrate/src/utils/ensureDatabaseExists.ts [hunk 2]
```diff
@@ -12,6 +12,31 @@ import execa from 'execa'
...
+  const dbLocation = getDbLocation(credentials)
+  const dbinfoFromCredentials = getDbinfoFromCredentials(credentials)
+  return {
+    name: activeDatasource.name,
+    dbLocation,
+    ...dbinfoFromCredentials,
+    url: activeDatasource.url.value,
+  }
+}
+
```
Comment (tomhoule) [inline src/packages/migrate/src/utils/ensureDatabaseExists.ts:39]: I am wondering, does this already work with MSSQL? (it has a really "unique" connection string syntax)

Comment (Jolg42) [reply src/packages/migrate/src/utils/ensureDatabaseExists.ts:39]: I don't know 🤷

File: src/packages/migrate/src/__tests__/MigrateDev.test.ts (hunk 1 above)
Comment (Jolg42) [reply src/packages/migrate/src/__tests__/MigrateDev.test.ts:597]: White space in snapshot is a bit out of control, if you know how to make them behave it's something we can improve in general in tests.

**Pull request merged**